helix-core/src/test/java/org/apache/helix/integration/task/TestQuotaBasedScheduling.java

```diff
@@ -135,6 +141,7 @@ public void testSchedulingWithoutQuota() throws InterruptedException {
     Workflow.Builder workflowBuilder = new Workflow.Builder(workflowName);
     WorkflowConfig.Builder configBuilder = new WorkflowConfig.Builder(workflowName);
     configBuilder.setAllowOverlapJobAssignment(true);
+    configBuilder.setJobPurgeInterval(-1);
```

Comment:
Are you suggest we remove these changes one by one if later we fix the purge issue?
@NealSun96 , what is the ETA of your change, please? I think removing these changes will be troublesome. Unless we separate them into a single change and revert it completely later.

Reply:
It's scheduled this sprint, and I can finish it as fast as mid-next week